gnomad/utils/annotations.py

```diff
@@ -1138,3 +1138,67 @@ def hemi_expr(
         # mt.GT[0] is alternate allele
         gt.is_haploid() & (sex_expr == male_str) & (gt[0] == 1),
     )
+
+
+def variant_report(
+    table_to_parse: hl.Table,
+    variant: str,
+    output_path: str,
+    template: str = "",
+    ref_input: str = "GRCh38",
+) -> None:
+    """
+    Thise code reads in
+    1) A reference table to parse containing the variant of interest and its VRS Annotations
+    2) A string of the Chr, Pos, Ref, and Alt of a Variant
+    3) A GA4GH-VRS JSON Template
+
+    and reports back a JSON containing the GA4GH-VRS Information of the variant
+
+    prints it and reports it to the desired location
+
+    """
+    import json
+
+    chr_in, pos_in, ref_in, alt_in = variant.split("-")
+    table_filtered = table_to_parse.filter(
+        table_to_parse.locus
+        == hl.locus(contig=chr_in, pos=int(pos_in), reference_genome=ref_input)
+    )
+    table_filtered = table_filtered.filter(table_filtered.alleles == [ref_in, alt_in])
+
+    if table_filtered.count() != 1:
+        raise ValueError(
+            "Error: can only work with one variant for this code, 0 or multiple"
+            " returned"
+        )
+
+    with hl.utils.hadoop_open(
+        "gs://gnomad-vrs-io-finals/chromosome_seqID_dictionary.json", "r"
+    ) as f:
+        chr_reread = json.load(f)
+
+    with hl.utils.hadoop_open(
+        "gs://gnomad-vrs-io-finals/example_schema.json", "r"
+    ) as f:
+        model_schema = json.load(f)
```

Comment:
for simplicity, transparency in the code, and flexibilty for making future changes, i think it actually would be best to build the json from scratch